static/js/components/person-projects.js

```diff
@@ -0,0 +1,52 @@
+import fetchPublicData from "./whimsy-loader.js";
+const projects = await fetchPublicData('public_ldap_projects.json');
+var projectsById = {};
+
+{
+  Object.keys(projects.projects).forEach(project => {
+    projects.projects[project].members.forEach(m => {
+      if (!projectsById[m]) {
+        projectsById[m] = {};
+      }
+      projectsById[m][project] = { pmc: false };
+    })
+    projects.projects[project].owners.forEach(m => {
+      if (!projectsById[m]) {
+        projectsById[m] = {};
+      }
+      projectsById[m][project] = { pmc: true };
+    })
+  })
+}
+
+// some projects websites have names that
+// differ from their Whimsy name
+const projectToWebsite = {
```

Comment:
See also: https://whimsy.apache.org/docs/hardcoded.cgi which leads to https://github.com/apache/whimsy/blob/master/lib/whimsy/asf/committee.rb#L71 for Whimsy's full list of group name mappings.

Reply:
Ok thanks - so far considering there's just one such exception, relying on hardcoded things that can move unexpectedly feels risky. On https://whimsy.apache.org/docs/hardcoded.cgi , the `lib/whimsy/asf/site.rb`link for example became invalid.